.ci/release.sh

```diff
@@ -25,7 +25,7 @@ java -version
 publishArg=''
 if [[ "$dry_run" == "false" ]] ; then
     echo "--- Build and publish the release :package:"
-    publishArg='publishToSonatype closeAndReleaseStagingRepository'
```

Comment:
From the [Release notes](https://github.com/gradle-nexus/publish-plugin/releases/tag/v2.0.0):

> Backward incompatible changes
> closeAndReleaseStagingRepository has been renamed to closeAndReleaseStagingRepositories for consistency

Rest should be fine, I'll check the snapshot workflow after this PR is merged